rocky/tests/objects/test_objects_detail.py

```diff
@@ -63,6 +84,33 @@ def test_ooi_detail(
     assertContains(response, "Completed")
     assertContains(response, "Created date")
     assertContains(response, "9, 2022, 11:53 a.m.")
+    assertNotContains(response, "Question")
+    assertNotContains(response, "Rendered Question Form")
+
+
+def test_question_detail(
```

Comment:
You might want to test one of the error paths (500s) or the created response (201)

Reply:
Done!